collect_app/src/main/AndroidManifest.xml

```diff
@@ -301,6 +297,11 @@ the specific language governing permissions and limitations under the License.
             android:name="com.google.android.maps"
             android:required="false" />
 
+        <activity
+            android:name=".location.GeoActivity"
+            android:configChanges="orientation"
+            android:exported="true"
```

Comment:
@jknightco why is this set to true? I think it would be better not to expose this externally unless absolutely necessary.